Changes

```diff
@@ -1,4 +1,5 @@
-{{$NEXT}}
+{{$NEXT}} 2019-09-11 18:16:14 Cameroon/Yaounde
```

Comment:
This line is just a place holder, it will be replaced by Dist::Zilla when a release is created, and it will contain the time and location of the machine doing the release. Please revert it. 

Also, I think your date is slightly wrong. ;)